python/sglang/srt/layers/fused_moe/fused_moe.py

```diff
@@ -1,19 +1,19 @@
-# Adapted from
-# https://github.com/vllm-project/vllm/tree/v0.5.4/vllm/model_executor/layers/fused_moe
+# Adapted from https://github.com/vllm-project/vllm/tree/v0.5.5/vllm/model_executor/layers/fused_moe
```

Comment:
is this 100% copy or are there any modifications?

Reply:
Since this PR didn't change the fused moe layer, I think we can revert the first commit. The moe change will be involved in https://github.com/sgl-project/sglang/pull/1305.

Reply:
make sense